src/NuGet.Core/NuGet.Packaging/Signing/SignManifest.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NuGet.Versioning;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents the XML manifest containing a set of files and hashes that can be signed.
+    /// </summary>
+    public class SignManifest
```

Comment:
Consider adding a way to serialize this, which can then be written to disk or provide a byte stream that can be inserted into the `ZipArchive`.

Reply:
Good point, I'll add a Write method